src/librustc/session/config.rs

```diff
@@ -1259,7 +1259,7 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
         "prints the LLVM optimization passes being run"),
     ast_json: bool = (false, parse_bool, [UNTRACKED],
         "print the AST as JSON and halt"),
-    threads: Option<usize> = (None, parse_opt_uint, [UNTRACKED],
+    threads: Option<usize> = (Some(1), parse_opt_uint, [UNTRACKED],
```

Comment:
It seems a bit odd to have this be `Option<usize>` when the default isn't `None`, could this instead change `threads_from_count`, and a comment could be included why it's defaulted to one?

Reply:
Sure, yeah.